app/controllers/spotlight/catalog_controller.rb

```diff
@@ -131,7 +131,7 @@ def attach_breadcrumbs
         add_breadcrumb t(:'spotlight.exhibits.breadcrumb', title: @exhibit.title), exhibit_root_path(@exhibit, q: '')
       else
         # When not on the admin page, get the translated value for the "Home" breadcrumb
-        add_breadcrumb t(:'spotlight.curation.nav.home', title: @exhibit.title), exhibit_root_path(@exhibit, q: '')
```

Comment:
The comment above this code suggested it was a necessary workaround, at least at some point in time. We should check if it's still an issue, or if it's fixed, also fix this on line 131, remove the comment, and pin the dependencies as appropriate.